apollo-api/src/main/java/com/apollostack/android/PostBody.java

```diff
@@ -0,0 +1,24 @@
+package com.apollostack.android;
+
+import com.apollostack.api.GraphQLOperation;
+import com.apollostack.api.GraphQLQuery;
+
+import javax.annotation.Nullable;
+
+public class PostBody<T extends GraphQLOperation.Variables> {
```

Comment:
I see, ok can we then make this class package protected and create it in ApolloRequestBodyConverter and pass it to JsonAdapter?